mne/io/edf/tests/test_edf.py

```diff
@@ -408,8 +409,8 @@ def test_bdf_multiple_annotation_channels():
 def test_edf_lowpass_zero():
     """Test if a lowpass filter of 0Hz is mapped to the Nyquist frequency."""
     raw = read_raw_edf(edf_stim_resamp_path)
-    assert raw.ch_names[100] == 'EEG LDAMT_01-REF'
-    assert len(raw.ch_names[100]) > 15
+    assert raw.ch_names[100] == 'LDAMT_01-REF'
+    assert len(raw.ch_names[100]) > 11
```

Comment:
```suggestion
    assert raw.ch_names[100] == 'LDAMT_01-REF'
```
we've already asserted exact match of the name, additionally asserting string length is redundant